deploy/test-environments/fleet_api/src/agents_enrolled.py

```diff
@@ -12,6 +13,21 @@
 TIMEOUT = 600
 
 
+class AgentPolicyEnrolled:
+    """
+    Class to represent the details of an enrolled agent.
+    """
+
+    def __init__(self, count: int, tags: list[str]) -> None:
+        """
+        Args:
+            count (int): Number of agents to be enrolled.
+            tags (list[str]): Tags of the agent.
+        """
+        self.count = count
+        self.tags = tags
+
+
```

Comment:
It appears that this class is exclusively utilized for data. Therefore, instead of employing a standard class, you could consider implementing a [dataclass](https://docs.python.org/3/library/dataclasses.html)

We already have an implementation example available [here](https://github.com/elastic/cloudbeat/blob/main/tests/product/tests/data/eks_test_case.py)